e2e/charts/newrelic-infrastructure/templates/deployment.yaml

```diff
@@ -34,17 +34,20 @@ spec:
       {{- end }}
       containers:
         - name: forwarder
-          image: newrelic/k8s-events-forwarder:1.20.5
-          imagePullPolicy: IfNotPresent
+          image: {{ .Values.forwarderImage.repository }}:{{ .Values.forwarderImage.tag }}
+          imagePullPolicy: {{ .Values.forwarderImage.pullPolicy }}
           securityContext:
-            privileged: false
+            {{- if .Values.privileged }}
+            privileged: true
+            {{- else }}
```

Comment:
Ditto, I'm not sure scrapers care about the `privileged` flag.

Reply:
it does not, you are right